src/annotate-source.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright 2017 Shape Security, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+function insertInto(annotations, index, text, afterExisting) {
+  for (let i = 0; i < annotations.length; ++i) {
+    if (annotations[i].index >= index) {
+      if (afterExisting) {
+        while (i < annotations.length && annotations[i].index === index) {
+          ++i;
+        }
+      }
+
+      annotations.splice(i, 0, { index, text });
+      return;
+    }
+  }
+  annotations.push({ index, text });
+}
+
+class Info {
+  constructor() {
+    this.declares = [];
+    this.reads = [];
+    this.writes = [];
+    this.scopes = [];
+  }
+}
+
+/* This should work, but babel.
```

Comment:
Link to an open issue so we know when we can uncomment this?

Reply:
I'm pretty sure it's wontfix on their part; there's no way to do it safely all the time.

I guess a better solution is to change the babel config to only do the `import` -> `require` transformation, since we're not supporting node < 4. Maybe it's time we switched to [babel-preset-env](https://github.com/babel/babel-preset-env)?

Reply:
Actually, babel-preset-env [seems to think](https://github.com/babel/babel-preset-env/blob/be7a61c368149cd7e9d4202ecec6f71c56f2eb77/data/plugins.json#L48) node 4 does not support classes. (Apparently the kangax table has node 4 [marked](https://kangax.github.io/compat-table/es6/#node4) as supporting classes only in strict mode, though in my tests this seems to be false, and babel-preset-env just interprets this as "no support for classes".)

Tempted to override babel-preset-env to manually disable the class transform. Thoughts?

Reply:
PR babel-preset-env, obviously.

Reply:
babel-preset-env pulls from the kangax table.

I could PR the kangax table, but I suspect they do have a reason for claiming classes aren't supported.

Reply:
> Tempted to override babel-preset-env to manually disable the class transform. 

Went with this. Tests pass on node 4, so whatever.

This PR should be ready to go.